### PR TITLE
[Enhancement] (Map Control) use lazy tippy to improve map legend rendering perf

### PR DESCRIPTION
**Pull request by igorDykhta**
* use lazy tippy to improve map legend rendering perf

Signed-off-by: Shan He <heshan0131@gmail.com>

### CODE REVIEW

File: test/browser/components/map/map-control-test.js [hunk 1]
```diff
@@ -165,15 +165,17 @@ test('MapControlFactory - click options', t => {
     .at(0)
     .simulate('click');
 
-  t.equal(wrapper.find(MapLegend).length, 1, 'should render MapLegend');
+  // TODO the test doesn't work with LazyTippy
+  // t.equal(wrapper.find(MapLegend).length, 1, 'should render MapLegend');
```
Comment (igorDykhta) [inline test/browser/components/map/map-control-test.js:169]: @heshan0131 Looks like this test doesn't work with LazyTippy. mounted doesn't change to true during the test run.

File: src/components/map/lazy-tippy.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+// @ts-nocheck
+import React, {useState, forwardRef} from 'react';
+import Tippy from '@tippyjs/react/headless';
+
+/**
+ * Lazy mounting tippy content
+ * https://gist.github.com/atomiks/520f4b0c7b537202a23a3059d4eec908
+ */
+// eslint-disable-next-line react/display-name
+const LazyTippy = forwardRef((props, ref) => {
```
Comment (heshan0131) [inline src/components/map/lazy-tippy.tsx:10]: We use a isTest util to mount lazy tippy in test for testing
```js
import {isTest} from 'utils/utils';

const isTestEnv = isTest();

const LazyTippy = forwardRef((props, ref) => {
  // Mount in test env for easier testing
  const [mounted, setMounted] = useState(isTestEnv);
```

File: test/browser/components/map/map-control-test.js (hunk 1 above)
Comment (heshan0131) [reply test/browser/components/map/map-control-test.js:169]: see comment in lazy-tippy

Comment (heshan0131) [reply test/browser/components/map/map-control-test.js:169]: https://github.com/keplergl/kepler.gl/pull/1924/files#r946541747

**Review by heshan0131 — APPROVED** [review]

**Pull request merged**